src/models/kernels.hh

```diff
@@ -165,9 +165,8 @@ namespace rascal {
         } else if (this->zeta == 3) {
           kernel = kernel.array().cube();
         } else {
-          kernel = kernel.unaryExpr([zeta = this->zeta](double v) {
-            return math::pow(v, zeta);
-          });
+          kernel = kernel.unaryExpr(
+              [zeta = this->zeta](double v) { return math::pow(v, zeta); });
```

Comment:
This reformatting is strange, I had already run clang-format. Was this automatic? If so, which version of clang-format do you have installed?

Reply:
I have clang-format 8

Reply:
Ok. I'll look into this.